README.md

```diff
@@ -8,12 +8,13 @@ All content must be freely licensed.
 
 ### Code
 * `skins` MIT by rubenwardy
-* `snow` MIT by LoneWolf, rubenwardy, Thomas-S, and GreenXenith
-* `winterize` MIT by LoneWolf
+* `snow` MIT by LoneWolfHT, rubenwardy, Thomas-S, and GreenXenith
+* `winterize` MIT by LoneWolfHT
 
 ### Media
+
 * `Santa` CC BY-SA 3.0 by jordan4ibanez
```

Comment:
This line doesn't apply to both the skins in common now, and can be removed(?)

Reply:
Yes, this should be changed to just ```* `Santa` ```.

Reply:
Done